UKFTractography.cmake

```
@@ -20,18 +19,16 @@ include(${ITK_USE_FILE})
 #-----------------------------------------------------------------------------
 set(VTK_FOUND OFF)
 find_package(VTK COMPONENTS
```

```suggestion
find_package(VTK 9 COMPONENTS
```

This will make thing unambiguous